Makefile

```diff
@@ -57,6 +57,14 @@ BABEL := $(NODE_LOCAL_BIN)/babel
 ESLINT := $(NODE_LOCAL_BIN)/eslint
 FLAKE8 := $(NODE_LOCAL_BIN)/flake8
 
+ifdef DEV_XPI
+LOOP_XPI_DATE=`date +"%Y%m%d%H%M"`
```

Comment:
The number rules are complex, but I think that dashes aren't allowed. I don't think we really need them here either tbh.